src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplSearchSteps.java

```diff
@@ -1,52 +1,54 @@
 package gov.healthit.chpl.aqa.stepDefinitions;
-
 import cucumber.api.java.en.Given;
 import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
 import org.apache.commons.lang3.StringUtils;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertTrue;
 
+import static org.testng.Assert.assertTrue;
 import gov.healthit.chpl.aqa.pageObjects.ListingDetailsPage;
 import gov.healthit.chpl.aqa.pageObjects.SearchPage;
-
 /**
- * Class DisclosureUrlSteps definition.
+ * Class DeleteDupListingsSteps definition.
  */
-public class DisclosureUrlSteps {
 
+public class ChplSearchSteps {
     private WebDriver driver;
     private static final int TIMEOUT = 30;
     private String url = System.getProperty("url");
 
     /**
      * Constructor creates new driver.
      */
-    public DisclosureUrlSteps() {
+    public ChplSearchSteps() {
         driver = Hooks.getDriver();
         if (StringUtils.isEmpty(url)) {
             url = "http://localhost:3000/";
-        }
-   }
+           }
+    }
+    /**
+     * Get user to CHPL search page.
+     */
+    @Given("^I am on CHPL search page$")
+    public void iAamOnCHPLSearchPage() {
+        driver.get(url);
+    }
```

Comment:
Should have something here that waits until something on the page exists, otherwise the next step might start before it can work